WordPressVIPMinimum/Sniffs/Files/IncludingFileSniff.php

```diff
@@ -123,7 +123,9 @@ public function process( File $phpcsFile, $stackPtr ) {
 				return;
 			}
 
-			if ( 1 === preg_match( '/^[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*$/', $tokens[ $nextToken ]['content'] ) ) {
+			$nextNextToken = $phpcsFile->findNext( Tokens::$emptyTokens, ( $nextToken + 1 ), null, true, null, true );
+
+			if ( 1 === preg_match( '/^[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*$/', $tokens[ $nextToken ]['content'] ) && T_OPEN_PARENTHESIS !== $tokens[ $nextNextToken ]['code'] ) {
```

Comment:
Which test case goes with this check?

Reply:
`include ( MY_CONSTANT . "my_file.php" ); // Custom constant.`